play/src/components/RoundTextIcon.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Button } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import SelectedAnswerImage from '../img/selectedAnswerImage.svg';
+import UnselectedAnswerImage from '../img/unselectedAnswerImage.svg';
+import CorrectAnswerImage from '../img/correctAnswerImage.svg';
+
+interface RoundTextIconProps {
+  answerStatus: 'default' | 'correct' | 'selected';
+  index: number;
+  answerText: string;
+  isSubmitted: boolean;
+  handleSelectAnswer: (index: number) => void;
+}
+
+const RoundTextIcon = ({
+  answerStatus,
+  index,
+  answerText,
+  isSubmitted,
+  handleSelectAnswer,
+} : RoundTextIconProps) => {
+  const classes = useStyles(isSubmitted)();
+  const letterCode = 'A'.charCodeAt(0) + index; 
+
+  const imageMap = {
+    'default': UnselectedAnswerImage,
+    'correct': CorrectAnswerImage,
+    'selected': SelectedAnswerImage,
+  }
+
+  const styleMap = {
+    'default': classes.buttonDefault,
+    'correct': classes.buttonCorrect,
+    'selected': classes.buttonSelected,
+  }
+ 
+  return (
+    <div className={classes.container}>
+      <Button
+        className={`${classes.buttonBase} ${styleMap[answerStatus]}`}
+        onClick={() => handleSelectAnswer(index)}
+        disabled={isSubmitted || answerStatus === 'correct'}
+        variant="text"
+      >
+        <div className={`${classes.letterText} ${isSubmitted || answerStatus === 'correct' ? classes.letterTextSubmitted : ''}`}>  
+          {`${String.fromCharCode(letterCode)}.`} 
+        </div>
+        <div className={classes.answerText}>  {answerText} </div>
```

Comment:
Any reason for extra spaces before and after?

Reply:
fixed via prettier